pkg/migrations/messages/20240419172419_remove-unused-indexes-pt-1.down.sql

```diff
@@ -0,0 +1,5 @@
+SET statement_timeout = 0;
+
+--bun:split
+
+CREATE INDEX CONCURRENTLY IF NOT EXISTS message_pubsubTopic_idx ON public.message (pubsubTopic);
```

Comment:
Without CONCURRENTLY, does the server start up extremely slowly?

Reply:
It would also take a lock of the entire table and block reads

Reply:
I'm having problems actually running concurrent index drops in a migration (for some reason it is trying to run the migration in a tx, which is not allowed). I might just make this non-concurrent but manually drop the index concurrently in prod first.